hail/src/main/scala/is/hail/io/vcf/LoadVCF.scala

```diff
@@ -1138,9 +1138,12 @@ object LoadVCF {
     callFields: Set[String],
     entryFloatType: TNumeric,
     lines: Array[String],
-    arrayElementsRequired: Boolean = true
+    arrayElementsRequired: Boolean = true,
+    repairHeader: Boolean = true
   ): VCFHeaderInfo = {
     val codec = new htsjdk.variant.vcf.VCFCodec()
+    if (!repairHeader)
```

Comment:
I'd be happier just turning this off in all cases without any parameterization. This has caused nothing but trouble in the past!

Reply:
it certainly couldn't have been easy to find all the places we thread the parameters through, though! Nice work.

Reply:
@tpoterba I'm happy to remove the conditional, if you are ok with that, that simplifies things obviously.

Reply:
yes, absolutely.